.github/workflows/MERGE_MAIN_TO_DEVELOP.yml

```diff
@@ -42,10 +42,10 @@ jobs:
           secret/data/products/desktop-modeler/ci/slack_integration SLACK_CHANNEL_ID;
           secret/data/products/desktop-modeler/ci/slack_integration SLACK_BOT_TOKEN;
     - name: Notify on Slack if merge fails
-      uses: slackapi/slack-github-action@v1
-      if: failure()
```

Comment:
This is redundant as the job contains the failure condition already.

Reply:
Good find!